service-commands/src/libs.js

```diff
@@ -89,6 +89,9 @@ function LibsWrapper (walletIndex = 0) {
     )
 
     const walletAddress = config.get('data_wallets')[walletIndex]
+    this.walletAddress = walletAddress
+    this.walletIndex = walletIndex
```

Comment:
are these used anywhere?

Reply:
not used currently anywhere, but would be helpful in refactoring mad dog